docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -1007,6 +1013,18 @@ public class MyConfiguration {
 ----
 ====
 
+[[loadbalancer-lifecycle]]
+=== Spring Cloud LoadBalancer Lifecycle
+
+One type of beans that it may be useful to register using <<custom-loadbalancer-configuration,Custom LoadBalancer configuration>> is `LoadBalancerLifecycle`.
+
+The `LoadBalancerLifecycle` beans provide callback `onStart(Request<RC> request)` and `onComplete(CompletionContext<RES, T> completionContext)` methods that you should implement to specify what actions should take place before and after load-balancing.
+
+`onStart(Request<RC> request)` takes as a `Request` object as a parameter. It contains data that will be used to select an appropriate instance, including the downstream client request and <<spring-cloud-loadbalancer-hints,hint>>. To the `onComplete(CompletionContext<RES, T> completionContext)` method, on the other hand, a `CompletionContext` object is provided that contains the LoadBalancer `Response`, including the selected service instance, the `Status` of the request executed against that service instance and, if available, the response returned to the downstream client and, if an exception has occurred, the corresponding `Throwable`.
```

Comment:
Change to

`onStart(Request<RC> request)` takes a `Request` object as a parameter. It contains data that is used to select an appropriate instance, including the downstream client request and <<spring-cloud-loadbalancer-hints,hint>>. On the other hand, a `CompletionContext` object is provided to the `onComplete(CompletionContext<RES, T> completionContext)` method. It contains the `LoadBalancer` `Response`, including the selected service instance, the `Status` of the request executed against that service instance and (if available) the response returned to the downstream client, and (if an exception has occurred) the corresponding `Throwable`.